.github/workflows/default.yaml

```diff
@@ -44,9 +44,8 @@ jobs:
 
       - run: ./bootstrap.sh
       - run: ./configure --with-openssl
-      - run: make -j2
```

Comment:
While "make install" may trigger "make all" or equivalent, we should not remove this step because we do not run "make install". We run "sudo make install". We should run "make [all]" without sudo because that is how most folks will run it (and because privileged access  is not supposed to be necessary for "make all", and because we do not bootstrap and ./configure under sudo, etc.).

Please do not forget to update PR description after restoring this command.

Reply:
Doh, you are right. It doesn't make much of a difference due to container isolation but it is poor practice, I'm rolling it back. I'll just use nproc instead of 2 as it might under- or over-utilize resources